addons/website_livechat/views/website_livechat.xml

```diff
@@ -3,6 +3,7 @@
     <data>
         <template id="assets_backend" inherit_id="website.assets_backend" name="website_livechat assets backend">
             <xpath expr="." position="inside">
+                <script type="text/javascript" src="/website_livechat/static/src/models/messaging_notification_handler/messaging_notification_handler.js"/>
```

Comment:
This override will be removed in the PR from XDU. You should put the new file into the `assets.xml` override.